packages/grid/x-data-grid-premium/src/hooks/features/export/gridExcelExportInterface.ts

```diff
@@ -16,7 +16,7 @@ export interface GridExceljsProcessInput {
 }
 
 export interface ColumnsStylesInterface {
```

Comment:
Off-topic. We missed the Grid prefix export on this one

```suggestion
export interface GridColumnsStylesInterface {
````